loops-nate-arnold.js

```diff
@@ -0,0 +1,39 @@
+// Logging values with for loops
+   
+// -Write a for loop that logs each number from 1 - 20.
+    for(let i=1; i<21; i++){
+      console.log(i)
+    }
+// -Write a for loop that logs the result of each number from 1 - 20 tripled.
+    var arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
+
+    for(let i=0; i<arr.length; i++){
+        console.log(arr[i] * 3)
+      }
+// -Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
```

Comment:
The current for loop is evaluating if the value at any index is not equal to 0, 
`if(arr[i] !== (0)){`
which will be true for everything in the assigned array except the index that has a value of 0. 
`var arr = [0, "ODD", 2, "ODD", 4, "ODD", 6, "ODD", 8, "ODD", 10, "ODD", 12, "ODD", 14, "ODD", 16, "ODD", 18, "ODD", 20]`
Therefore, the output displayed starts at the index that is true.
ODD
2
ODD
4
ODD
6
ODD
8

So I recommend using the approach for the question on Line 7. So you can refactor the lines 4-6 to give you the expected output and then introduce some conditional statements for the odd and even conditions.
for(let i = 1; i < 21; i++){
  if(i % 2 !== 0) {
    console.log("ODD")
  } else {
    console.log(i)
  }
}